examples/run_all.sh

```diff
@@ -0,0 +1,3 @@
+for EXAMPLE in *.py
```

Comment:
Saw this?
https://github.com/karlnapf/kernel_hmc/blob/master/kernel_hmc/examples/run_all.sh

Allows to use matplotlib on travis